packages/app/src/templates/components/template-options/template-options.component.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
```

Comment:
(adding it here as it is easier to have a thread)

I cannot add any new template. Tested on MuditaOS 1.2.0 - do you think that is the reason?

https://user-images.githubusercontent.com/41268731/173573155-23eaad02-c81d-4855-83ea-97c3aaa1037b.mov

Reply:
You probably should use the latest daily build of MuditaOS. The official release doesn't contains those changes